crates/napi/scripts/generate-types.ts

```diff
@@ -0,0 +1,264 @@
+import { readFile, writeFile } from "node:fs/promises";
+import path from "node:path";
+import { Edit, kind, Lang, parseAsync } from "../index";
+import { Rule } from "../manual";
+import { NodeTypeSchema } from "../types/node-types";
+
+const languagesNodeTypesUrls = {
+  [Lang.JavaScript]:
+    "https://raw.githubusercontent.com/tree-sitter/tree-sitter-javascript/refs/heads/master/src/node-types.json",
```

Comment:
Is it possible to use git tag's node-types version instead of the head? ast-grep is using specific version on crates so it is not always safe to assume the git head is matching types.